src/style.css

```diff
@@ -1 +1,56 @@
-/* add styles here */
+/* #region page */
+html {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  margin: 0;
+}
```

Comment:
It's good practice to include a fallback font family for better cross-browser compatibility. For example, `font-family: 'YourFont', sans-serif;`